config/initializers/disable_active_storage_analyzer.rb

```diff
@@ -0,0 +1,5 @@
+# We don't want to fill up the queue with jobs that hammer the database
+# (and extract metadata we don't use). Ref https://stackoverflow.com/a/74879360.
+Rails.application.config.to_prepare do
+  ActiveStorage::Attachment.skip_callback(:commit, :after, :analyze_blob_later)
```

Comment:
Should we set empty analyzers/previewers in the Rails config instead? As of https://github.com/rails/rails/pull/37773 that should no longer create no-op queued jobs.

Reply:
Nice! I was mistaking the `PurgeJob` for `AnalyzeJob` when I thought that didn't work still. Much better!